lms/services/lti_grading/_v13.py

```diff
@@ -56,13 +59,26 @@ def record_result(self, grading_id, score=None, pre_record_hook=None):
         if pre_record_hook:
             payload = pre_record_hook(score=score, request_body=payload)
 
-        return self._ltia_service.request(
-            "POST",
-            self._service_url(self.line_item_url, "/scores"),
-            scopes=self.LTIA_SCOPES,
-            json=payload,
-            headers={"Content-Type": "application/vnd.ims.lis.v1.score+json"},
-        )
+        try:
+            return self._ltia_service.request(
+                "POST",
+                self._service_url(self.line_item_url, "/scores"),
+                scopes=self.LTIA_SCOPES,
+                json=payload,
+                headers={"Content-Type": "application/vnd.ims.lis.v1.score+json"},
+            ).json()
```

Comment:
This is an extra change, now we return `.json()` not just the response itself.

That seems more inline with the rest of the service.

Reply:
:+1: